mini google assistant.py

```diff
@@ -0,0 +1,97 @@
+import speech_recognition as sr
+import pyttsx3
+import pywhatkit
+import datetime
+import pyjokes
+import cv2
```

Comment:
now delete this file:

![image](https://user-images.githubusercontent.com/43691873/114809589-1407af00-9dc8-11eb-9eb2-6dd4ca9b2919.png)

Reply:
@kaustubhgupta Sir, Done!